message_app/user_sign_up.py

```diff
@@ -26,8 +26,8 @@ def api_user_signup():
             raise Exception(session["error"])
         else:
             # Hash the password
-            password, salt = hashing(password)
-            new_user = User(username=username, email=email, password_hash=password, password_salt=salt)
+            password_hash = hash_pw(password)
+            new_user = User(username=username, email=email, password_hash=password)
```

Comment:
Shouldn't this be `password_hash=password_hash`?

Reply:
yes